examples/send-plugin-message/README.md

```diff
@@ -0,0 +1,18 @@
+# How to send a plugin message to your html application
+This example demonstrates how to set up a presentation that sends plugin messages to your HTML application. Once received, these messages can be forwarded to your backend for processing.
+
+The setup consists of three components:
+
+1. **Presentation File (`bpfx`)**: This file supports sending plugin messages from the included widgets.
+2. **Script File (`pluginMessageTransfer.brs`)**: Captures the plugin messages and passes them to the HTML application.
```

Comment:
Probably best to mention "sends and receives the plugin messages between the BrightAuthor:connected Presentation"

Though, I realize this example does not "send and receive". We should update this to include sending and receiving. Please make this change. If this is not obvious what this means or how to do so, just let @mingjun7 @lherlein or I know and I'm sure we can quickly bring you up to speed.

Reply:
So right now this application receives plugin messages. Are you saying we should add in a way to send messages to Bacon as well? @bhastings1019

Reply:
Right - I'm thinking it may make sense to have a complete lifecycle of the how a plugin message can be sent and received. 

BAcon -> plugin -> JS

JS -> plugin -> BAcon

a block diagram would be helpful

Reply:
I realize this is an additional request. Let's file a ticket and discuss.

Reply:
Ticket created: https://brightsign.atlassian.net/browse/PE-550